src/import-data/ImportDataDestination.test.ts

```diff
@@ -399,43 +406,36 @@ describe('ImportDataDestination', () => {
 
       const accessControlNotice = screen.queryByText('Additional access controls', { exact: false });
 
-      expect(!!importNoticeHeader).toEqual(shouldShowAdditionalAccessControlNotice);
-      expect(!!accessControlNotice).toEqual(shouldShowAdditionalAccessControlNotice);
+      expect(!!importNoticeHeader).toEqual(displayExtraAccessControlNotice);
+      expect(!!accessControlNotice).toEqual(displayExtraAccessControlNotice);
     }
   );
 
   it.each([
     {
-      policies: [],
+      workspace: makeAzureWorkspace({ policies: [] }),
       shouldDisplayPolicies: false,
     },
     {
-      policies: [protectedDataPolicy],
+      workspace: makeAzureWorkspace({ policies: [protectedDataPolicy] }),
       shouldDisplayPolicies: true,
     },
   ] as {
-    policies: WorkspacePolicy[];
+    workspace: WorkspaceWrapper;
     shouldDisplayPolicies: boolean;
   }[])(
     'displays workspace policies on import if present on target workspace',
-    async ({ policies, shouldDisplayPolicies }) => {
+    async ({ workspace, shouldDisplayPolicies }) => {
       // Arrange
       const user = userEvent.setup();
-
+      const workspaceName = workspace.workspace.name;
       asMockedFn(canImportIntoWorkspace).mockReturnValue(true);
 
       setup({
         props: {
           importRequest: { type: 'pfb', url: new URL('https://example.com/path/to/file.pfb') },
         },
-        workspaces: [
-          makeAzureWorkspace({
-            workspace: {
-              name: 'workspace-name',
```

Comment:
I thought it made more sense to let the fixture provide the workspaceName, since it's not actually pertinent to the test case.

In doing so, I thought it also made sense to push the logic to create a workspace up to the test case declaration.